tests/bud/all-platform/Containerfile.default-arg

```diff
@@ -0,0 +1,2 @@
+ARG foo=alpine
+FROM $foo
```

Comment:
Does that really add a new test?  I see many tests using `ARG SAFEIMAGE` already.

Reply:
all of them do not have any default value set for `ARG` instead they all consume value from cli using `--build-arg`, this test verifies additional heading arg i.e default value and it was not supported before this PR.

Reply:
Line: https://github.com/containers/buildah/pull/4401/files#diff-d96798032784de488b331cd60b9420f3263b46da4e7b47f8968569a0e6f629beR699 added support for this